src/lista/repeticao/Ex100.java

```diff
@@ -0,0 +1,48 @@
+package lista.repeticao;
+
+import java.util.Scanner;
+
+public class Ex100 {
+    public static void main(String[] args) {
+        Scanner leia = new Scanner(System.in);
+        int maior_estoque = 0;
+        int estoque_bola, estoque_boneca, estoque_ursinho;
+        int bola = 80,
+                boneca = 100,
+                ursinho = 20;
+        int venda_bola = 20,
+                venda_boneca = 10,
+                venda_ursinho = 5;
+
+        estoque_bola = bola - venda_bola;
```

Comment:
Todo exercício dessa lista de repetição precisa de um loop (for, do while ou while). Aqui está faltando um loop. No enunciado não fica claro qual a condição de parada, mas podemos imaginar que a nossa condição vai ser quando o usuário digitar um nome do produto = "@".